rott/modexlib.c

```diff
@@ -209,9 +211,37 @@ void TurnOffTextCursor ( void )
 =
 ====================
 */
+
+static Uint64 next_time = 0;
+
+static void time_start(void)
+{
+	next_time = SDL_GetTicks() + VBLCOUNTER;
+}
+
+static Uint64 time_left(void)
+{
+	Uint64 now = SDL_GetTicks();
+
+	if (next_time <= now)
```

Comment:
Is there anything special we should do if `now > next_time + VBLCOUNTER`, i.e. if `WaitVBL()` hasn't been caled for more than one tic?

Reply:
I don't think so, but I'm not totally sure.